backend/test/acceptance/webhook.spec.js

```diff
@@ -42,18 +45,18 @@ describe('github', function () {
     let newGithubIssue
 
     beforeEach(function () {
-      newGithubIssue = fixtures.github.issues.create({
-        number: 42,
-        updated_at: '2006-01-02T15:04:05.000Z'
-      })
+      newGithubIssue = fixtures.github.issues.create({ number: 42 })
       mockRequest.mockImplementationOnce(fixtures.leases.mocks.mergePatch())
     })
 
     afterEach(() => {
       jest.resetAllMocks()
     })
 
-    it('should handle valid github webhook', async function () {
+    it('should handle valid github webhook', async () => {
+      // do not enable that globally (e.g. in beforeEach) because express error handlers
+      // won't be called when fake timers are enabled.
+      jest.useFakeTimers().setSystemTime(now)
```

Comment:
I would prefer to do this in beforeAll and AfterAll or do the reset in finally.

Reply:
Enabling the fake timers by `beforeAll`/`afterAll` of the parent `describe` block would break the test `should error on invalid github webhook event` because the fake timers somehow mess with express's error handling and cause the test case to never complete.

As only the test `should handle valid github webhook` requires the fake timers I assumed it is sensible to enable/disable them on the lowest "layer" possible to prevent side effects with other and future tests.